tests/utils/test_env.py

```diff
@@ -77,6 +78,20 @@ def test_virtualenvs_with_spaces_in_their_path_work_as_expected(tmp_dir, manager
     assert venv.run("python", "-V", shell=True).startswith("Python")
 
 
+def test_venv_backup_exclusion(tmp_dir, manager):
```

Comment:
What about adding:

```python
@pytest.mark.skipif(sys.platform == "darwin", reason="requires darwin")
```

Then you can get rid of the if on line 86.

Reply:
Good call